packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -428,9 +412,9 @@ export const Stop = () => {
               <ItinerarySegmentBanner>
                 <ItineraryBadgeList>
                   <ItineraryBadgeListItem type="warning" icon={<StarFull color="warning" />}>
-                    <StyledText as="span" type="warning">
+                    <Text as="span" type="warning">
```

Comment:
Looks like this is bigger than it should

![image](https://github.com/kiwicom/orbit/assets/8863238/f54c8994-e056-42b4-9807-a365f1e32d82)

Reply:
Well spotted. It's a Story issue only, luckily. Fixed